zappa/cli.py

```diff
@@ -422,7 +422,7 @@ def schedule(self):
                 events = []
             keep_warm_rate = self.zappa_settings[self.api_stage].get('keep_warm_expression', "rate(5 minutes)")
             events.append({'name': 'zappa-keep-warm',
-                           'function': self.lambda_handler,
+                           'function': 'handler.heater',
```

Comment:
@Miserlou Since it's just a CW event, `lambda_handler` will be called no matter what. Then it will call `handler.heater` via Scheduled event branch. If I'm reading it correctly, it never was able to get to the web request handling code (i.e. to warm up application module import). Having said that, we now can import any modules in the `heater` method and probably even to call back into a user specified warmup function. 

I might be reading it wrong, of course.